.github/workflows/test.yml

```diff
@@ -232,10 +232,14 @@ jobs:
         - { name: 'GCC 5 (Ubuntu Xenial - 16.04)',    eval: 'CC=gcc-5 && CXX=g++-5',           build: 'release_strict' }
         - { name: 'GCC 6 (Debian Stretch)',           eval: 'CC=gcc-6 && CXX=g++-6',           build: 'release_strict' }
         - { name: 'GCC 7 (Ubuntu Bionic - 18.04)',    eval: 'CC=gcc-7 && CXX=g++-7',           build: ''               }
-        - { name: 'GCC 9 (Latest Release)',           eval: 'CC=gcc-9 && CXX=g++-9',           build: ''               }
         - { name: 'GCC 8 (Ubuntu Latest)',            eval: 'CC=gcc-8 && CXX=g++-8',           build: ''               }
+        - { name: 'GCC 9',                            eval: 'CC=gcc-9 && CXX=g++-9',           build: ''               }
```

Comment:
Could you add comments about which distro have these compilers as default (see other examples)?

Reply:
I looked up the defaults for all supported ubuntu and debian releases, turns out GCC 5 and clang 8/9 aren't actually used anywhere.